makefile.cargo

```diff
@@ -1,4 +1,4 @@
-CONFIGURE_FLAGS := --disable-jemalloc
+CONFIGURE_FLAGS := --enable-jemalloc
```

Comment:
``--enable-jemalloc` shouldn't have any effect, so I think this line can just be removed.